File: src/main/java/org/socialgame/dto/LoginRequest.java
package org.socialgame.dto;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class LoginRequest {
    // Getters y Setters obligatorios
    private String username;
    private String password;

    // Constructor vacío obligatorio para Jackson
    public LoginRequest() {
    }

    public LoginRequest(String username, String password) {
        this.username = username;
        this.password = password;
    }

}

File: src/main/kotlin/org/socialgame/dto/LoginRequest.kt
package org.socialgame.dto

class LoginRequest {
    // Getters y Setters obligatorios
    var username: String? = null
    var password: String? = null

    // Constructor vacío obligatorio para Jackson
    constructor()

    constructor(username: String?, password: String?) {
        this.username = username
        this.password = password
    }
}